packages/devtools_app/lib/src/app.dart

```diff
@@ -594,18 +597,34 @@ List<DevToolsScreen> defaultScreens({
 }) {
   return devtoolsScreens ??= <DevToolsScreen>[
     DevToolsScreen<void>(HomeScreen(sampleData: sampleData)),
-    DevToolsScreen<InspectorController>(
-      InspectorScreen(),
-      createController: (_) => InspectorController(
-        inspectorTree: InspectorTreeController(
-          gaId: InspectorScreenMetrics.summaryTreeGaId,
+    // TODO(https://github.com/flutter/devtools/issues/7860): Clean-up after
+    // Inspector V2 has been released.
+    if (FeatureFlags.inspectorV2)
+      DevToolsScreen<inspector_v2.InspectorController>(
+        inspector_v2.InspectorScreen(),
+        createController: (_) => inspector_v2.InspectorController(
+          inspectorTree: inspector_v2.InspectorTreeController(
+            gaId: InspectorScreenMetrics.summaryTreeGaId,
+          ),
+          detailsTree: inspector_v2.InspectorTreeController(
+            gaId: InspectorScreenMetrics.detailsTreeGaId,
+          ),
+          treeType: FlutterTreeType.widget,
         ),
-        detailsTree: InspectorTreeController(
-          gaId: InspectorScreenMetrics.detailsTreeGaId,
+      ),
+    if (!FeatureFlags.inspectorV2)
```

Comment:
use a ternary operator to make this more verbose (see example for logging screen below)